graphql/util.lua

```diff
@@ -115,6 +115,11 @@ local function coerceValue(node, schemaType, variables, opts)
             node.name.value, variables[node.name.value], schemaType.name
         ))
       end
+    elseif type(value) == 'table' and schemaType.__type == 'List' and
```

Comment:
I think something similar should be done for objects.

Reply:
I've added a simple test: seems that everything is alright

Reply:
Supported in separate commit